test/e2e/disruption.go

```diff
@@ -42,63 +42,123 @@ var _ = framework.KubeDescribe("DisruptionController [Feature:PodDisruptionbudge
 	})
 
 	It("should create a PodDisruptionBudget", func() {
-		pdb := policy.PodDisruptionBudget{
```

Comment:
(putting here because Github won't let me comment on a line that didn't change)

You should remove the `[Feature:PodDisruptionbudget]` tag on L34now that the feature is implemented, right?